pipelines/docker-build-multi-platform-oci-ta/README.md

```diff
@@ -215,6 +215,7 @@ This pipeline is pushed as a Tekton bundle to [quay.io](https://quay.io/reposito
 |YUM_REPOS_D_TARGET| Target path on the container in which yum repository files should be made available| /etc/yum.repos.d| |
 |caTrustConfigMapKey| The name of the key in the ConfigMap that contains the CA bundle data.| ca-bundle.crt| |
 |caTrustConfigMapName| The name of the ConfigMap to read CA bundle data from.| trusted-ca| |
+|image-digest| | | |
```

Comment:
Shall we not write some words about image digest ?

Reply:
There is no description for `image-url` either because we do not provide any description for these parameters in `task/sast-coverity-check/0.3/patch.yaml`.  The reason why the last two columns are empty is that we forgot to update `pipelines/docker-build/docker-build.yaml`:
```diff
--- a/pipelines/docker-build/docker-build.yaml
+++ b/pipelines/docker-build/docker-build.yaml
@@ -320,6 +320,8 @@ spec:
     params:
     - name: image-url
       value: $(tasks.build-image-index.results.IMAGE_URL)
+    - name: image-digest
+      value: $(tasks.build-image-index.results.IMAGE_DIGEST)
     - name: IMAGE
       value: $(params.output-image)
     - name: DOCKERFILE
```

Reply:
thanks @kdudka 
you are right, that was to fix the patch param of sast-coverity-check

Reply:
@kasemAlem You did not include the patch from my previous comment, which I think is needed.  The changes you done in `pipelines/docker-build/patch.yaml` do not seem to be correct.  We do **not** need to define new parameters of the `sast-coverity-check` task for this.  The task already has `image-url` and `image-digest` parameters defined in `task/sast-coverity-check/0.3/patch.yaml`.  We only need to set the values in the pipeline template(s).